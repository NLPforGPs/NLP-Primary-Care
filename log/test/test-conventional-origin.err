cpu-bind=MASK - bp1-gpu007, task 4294967295 4294967295 [0]: mask 0x202 set

CondaValueError: prefix already exists: /user/home/zu20361/.conda/envs/NLP_GP

[nltk_data] Downloading package wordnet to
[nltk_data]     /user/home/zu20361/nltk_data...
[nltk_data]   Package wordnet is already up-to-date!
[nltk_data] Downloading package punkt to
[nltk_data]     /user/home/zu20361/nltk_data...
[nltk_data]   Package punkt is already up-to-date!
Some weights of the model checkpoint at microsoft/BiomedNLP-PubMedBERT-base-uncased-abstract-fulltext were not used when initializing BertForSequenceClassification: ['cls.predictions.transform.dense.weight', 'cls.seq_relationship.bias', 'cls.predictions.bias', 'cls.seq_relationship.weight', 'cls.predictions.transform.LayerNorm.weight', 'cls.predictions.decoder.bias', 'cls.predictions.transform.LayerNorm.bias', 'cls.predictions.transform.dense.bias', 'cls.predictions.decoder.weight']
- This IS expected if you are initializing BertForSequenceClassification from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPreTraining model).
- This IS NOT expected if you are initializing BertForSequenceClassification from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model).
Some weights of BertForSequenceClassification were not initialized from the model checkpoint at microsoft/BiomedNLP-PubMedBERT-base-uncased-abstract-fulltext and are newly initialized: ['classifier.weight', 'classifier.bias']
You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.
2021-12-08 14:15:30,677 - /user/home/zu20361/NLP-Primary-Care/./run_plms.py[line:150] - INFO: Predicting...
2021-12-08 14:15:30,808 - /user/home/zu20361/.conda/envs/NLP_GP/lib/python3.9/site-packages/datasets/builder.py[line:377] - WARNING: Using custom data configuration default-a2d0abb6534a5099
0 examples [00:00, ? examples/s]2021-12-08 14:15:31,011 - /user/home/zu20361/.cache/huggingface/modules/datasets_modules/datasets/transcript_evaldataset/4c048f6bc1501180d0c57dc4d2b8c0d9958946e6a0943b3c5b80a90f1d04b330/transcript_evaldataset.py[line:59] - INFO: Generating examples...
                                  0%|          | 0/1 [00:00<?, ?it/s]100%|██████████| 1/1 [00:00<00:00, 563.90it/s]
2021-12-08 14:15:31,173 - /user/home/zu20361/NLP-Primary-Care/./run_plms.py[line:183] - INFO: conventional classifier...
2021-12-08 14:15:31,174 - /user/home/zu20361/NLP-Primary-Care/./run_plms.py[line:191] - INFO: id to label is {0: 'general', 1: 'blood', 2: 'digestive', 3: 'eye', 4: 'ear', 5: 'cardiovascular', 6: 'musculoskeletal', 7: 'neurological', 8: 'psychological', 9: 'respiratory', 10: 'skin', 11: 'endocrine', 12: 'urological', 13: 'pregnancy', 14: 'female', 15: 'male'}
  0%|          | 0/1 [00:00<?, ?ba/s]100%|██████████| 1/1 [00:01<00:00,  1.96s/ba]100%|██████████| 1/1 [00:01<00:00,  1.96s/ba]
  0%|          | 0/2 [00:00<?, ?ba/s] 50%|█████     | 1/2 [00:00<00:00,  1.27ba/s]100%|██████████| 2/2 [00:01<00:00,  1.95ba/s]100%|██████████| 2/2 [00:01<00:00,  1.81ba/s]

real	1m8.714s
user	0m23.085s
sys	0m10.242s
