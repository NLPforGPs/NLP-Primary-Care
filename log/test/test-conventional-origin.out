JOB ID: 243943
Working Directory: /user/home/zu20361/NLP-Primary-Care
Warning: you have pip-installed dependencies in your environment file, but you do not list pip itself as one of your conda dependencies.  Conda may not use the correct pip to install your packages, and they may end up in the wrong place.  Please add an explicit pip dependency.  I'm adding one for you, but still nagging you.
Start Time: Wed Dec 8 14:14:46 GMT 2021
loading codes from PE...
Downloading and preparing dataset description_dataset/default (download: Unknown size, generated: 2.07 MiB, post-processed: Unknown size, total: 2.07 MiB) to /user/home/zu20361/.cache/huggingface/datasets/description_dataset/default-a2d0abb6534a5099/0.0.0/4c048f6bc1501180d0c57dc4d2b8c0d9958946e6a0943b3c5b80a90f1d04b330...
Dataset description_dataset downloaded and prepared to /user/home/zu20361/.cache/huggingface/datasets/description_dataset/default-a2d0abb6534a5099/0.0.0/4c048f6bc1501180d0c57dc4d2b8c0d9958946e6a0943b3c5b80a90f1d04b330. Subsequent calls will reuse this data.
classification_report:
                 precision    recall  f1-score   support

          blood       0.50      0.62      0.56         8
 cardiovascular       0.53      0.81      0.64        32
      digestive       0.64      0.80      0.71        44
            ear       0.86      0.55      0.67        11
      endocrine       0.34      0.88      0.49        24
            eye       0.20      0.40      0.27         5
         female       0.43      0.71      0.54        14
        general       0.04      0.43      0.07        14
           male       0.45      0.71      0.56         7
musculoskeletal       0.73      0.83      0.78        65
   neurological       0.29      0.45      0.35        20
      pregnancy       0.40      0.18      0.25        11
  psychological       0.47      0.84      0.60        50
    respiratory       0.64      0.86      0.74        37
           skin       0.83      0.75      0.79        32
     urological       0.80      0.67      0.73        18

      micro avg       0.43      0.74      0.55       392
      macro avg       0.51      0.66      0.55       392
   weighted avg       0.57      0.74      0.62       392
    samples avg       0.46      0.76      0.55       392

f1_score 0.5455804356252656
roc_auc 0.5324575425015585
End Time: Wed Dec 8 14:15:54 GMT 2021
